active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,336 @@
+# Pager control
+
+## Background
+
+A UI pager is a common control that lets a user page through content, rather than scroll or pan. A common experience is to provide preview/next buttons, and a way to jump to a specific page.
+
+The absence of a standard WinUI pager control, last seen in Xaml as DataPager in Silverlight, has been a pain point in both WPF and UWP that has forced a variety of unfavorable workarounds for developers. Inclusion of this control in UWP would resolve an ecosystem gap and enable developers to efficiently deliver UI paging experiences in their applications. Through XAML Islands, it may also provide an opportunity to benefit WPF developers working with DataGrid or ListView. The scope of this control is not to provide a data paging solution at this time, but to start with UI paging as a foundation on top of which data paging may be later integrated.
+
+
+## Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+A pager control is a UI component that provides a standard interaction for pagination with a layout view (ListView, GridView, ItemsRepeater, DataGrid, etc.). A pager control is independent of the layout views and does not control any of the data being displayed in the layout view. 
+
+**Important APIs:** [PagerControl class](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.pagercontrol)
+
+This control can be configured to use one of three different display modes shown below:
+
+### ComboBox Mode
+![](images/pager_control_combo.jpg)
+
+### NumberBox Mode
+![](images/pager_control_number.jpg)
+
+### NumericalButtonPanel Mode 
+![](images/pager_control_button.jpg)
+
+## Is this the right control? 
+
+Use a **PagerControl** to create a navigation interface for an end user to page through content instead of scrolling or panning. 
+
+Scrolling through all content in a single view can make it harder for the end user to find specific information and it can lead to performance issues when the content needs to be refreshed. **PagerControl** can be used as an alternate solution so the user can consume a subset of the content without needing to scroll. They can then page through the rest of the content to find what they are looking for. 
```

Comment:
When should I use this vs ScrollViewer? This text makes it sound like I should never use a ScrollViewer, or there's some threshold number of items/pixels where I should transition to paging?

Reply:
I think the primary scenario for this control is data virtualization; you're trying to show 1000 items, but you don't want to put that load on the server/network in the typical case that the user only looks at the first 20.